__tests__/screens/send-bitcoin-screen.tsx

```diff
@@ -0,0 +1,342 @@
+    created_at: 1627328473,
+    date: "2021-07-26T19:41:13.000Z",
+    date_format: "Mon Jul 26 2021 14:41:13 GMT-0500",
+    date_nice_print: "hace 9 días",
+    description: "to Bitcoin",
+    fee: 0,
+    feeUsd: 0,
+    hash: null,
+    id: "60ff0fd95c95bd83528ad9b0",
+    isReceive: false,
+    pending: false,
+    text: "-$0.0040",
+    type: "on_us",
+    usd: 0.003969998828125,
+    username: "Bitcoin",
+  },
+  {
+    __typename: "Transaction",
+    amount: 1,
+    created_at: 1623602166,
+    date: "2021-06-13T16:36:06.000Z",
+    date_format: "Sun Jun 13 2021 11:36:06 GMT-0500",
+    date_nice_print: "hace 2 meses",
+    description: "whatIsBitcoin",
+    fee: 0,
+    feeUsd: 0,
+    hash: "a73651e6b7612f80ad2ff2676d1bf788218e95bce3d2cdf4e57cb5379e6592b1",
+    id: "60c633f6c6fbba6d6c74b352",
+    isReceive: true,
+    pending: false,
+    text: "$0.0004",
+    type: "on_us",
+    usd: 0.0003693000390625,
+    username: "BitcoinBeachMarketing",
+  },
+]
+
+const cache = new InMemoryCache()
+cache.writeQuery({
+  query: WALLET,
+  data: {
+    wallet: [
+      {
+        __typename: "Wallet",
+        balance: 117585,
+        currency: "BTC",
+        id: "BTC",
+        transactions: transactions,
+      },
+    ],
+  },
+})
+
+cache.writeQuery({
+  query: gql`
+    query nodeStats {
+      nodeStats {
+        id
+      }
+    }
+  `,
+  data: {
+    nodeStats: {
+      __typename: "NodeStats",
+      id: "",
+    },
+  },
+})
+
+cache.writeQuery({
+  query: QUERY_PRICE,
+  data: {
+    prices: [
+      {
+        __typename: "Price",
+        id: "1628106059",
+        o: 0.0003966375,
+      },
+    ],
+  },
+})
+
+cache.writeQuery({
+  query: gql`
+    query username {
+      me {
+        username
+      }
+    }
+  `,
+  data: {
+    me: {
+      username: "BitcoinBeach",
+    },
+  },
+})
+
+const payKeysendUsernameMocks = [
+  {
+    request: {
+      query: PAY_KEYSEND_USERNAME,
+      variables: {
+        amount: 25211,
+        destination: "",
+        username: "Bitcoin",
+      },
+    },
+    result: {
+      data: {
+        invoice: {
+          payKeysendUsername: "success",
+          __typename: "Invoice",
+        },
+      },
+    },
+  },
+]
+
+afterEach(cleanup)
+
+describe("SendBitcoinScreen", () => {
+  it("can render", () => {
+    render(
+      <MockedProvider cache={cache}>
+        <SendBitcoinScreen route={{ params: null }} />
+      </MockedProvider>,
+    )
+  })
+
+  it("has TextInputs", () => {
+    const { queryByPlaceholderText, queryByText } = render(
+      <MockedProvider cache={cache}>
+        <SendBitcoinScreen route={{ params: null }} />
+      </MockedProvider>,
+    )
+
+    expect(queryByText("0.00")).not.toBeNull()
+    expect(queryByPlaceholderText(translate("SendBitcoinScreen.input"))).not.toBeNull()
+    expect(queryByPlaceholderText(translate("SendBitcoinScreen.note"))).not.toBeNull()
+    expect(queryByPlaceholderText(translate("SendBitcoinScreen.fee"))).not.toBeNull()
```

Comment:
Should we assert values are `HTMLElement` here instead?

Reply:
The correct way to check type with jest seems to be to use [toBeInstanceOf](https://stackoverflow.com/a/62564851), but this returns the error `'HTMLElement' only refers to a type, but is being used as a value here.` It seems that these type errors should be caught at [compile time](https://stackoverflow.com/a/64113496), so are not needed in the unit test.